core/services/user_profile_service.py

```
@@ -0,0 +1,175 @@
+        self._session_user_ids[sid] = user_ids
+
+        if not user_ids or self.memory_sql_manager is None:
+            self._session_profiles[sid] = []
+            return []
+
+        try:
+            profiles = await self.memory_sql_manager.recall_user_profiles(
+                user_ids=user_ids,
+                memory_scope=memory_scope,
+            )
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)